src/opnsense/mvc/app/models/OPNsense/Trust/Trust.xml

```diff
@@ -0,0 +1,81 @@
+<model>
+    <mount>//OPNsense/Trust</mount>
+    <version>1.0.0</version>
+    <description>Trust/certificate manage</description>
+    <items>
+        <cas>
+            <ca type="ArrayField">
+                <refid type="TextField">
+                </refid>
+                <cauuid type="TextField">
+                </cauuid>
+                <descr type="TextField">
+                </descr>
+                <crt type="TextField">
+                </crt>
+                <prv type="TextField">
+                </prv>
+                <serial type="IntegerField">
+                </serial>
+            </ca>
+        </cas>
+        <certs>
+            <cert type="ArrayField">
+                <refid type="TextField">
+                </refid>
+                <cauuid type="TextField">
+                </cauuid>
+                <descr type="TextField">
+                </descr>
+                <crt type="TextField">
+                </crt>
+                <prv type="TextField">
+                </prv>
+                <csr type="TextField">
+                </csr>
+            </cert>
+        </certs>
+        <crls>
+            <crl type="ArrayField">
+                <refid type="TextField">
+                </refid>
+                <cauuid type="TextField">
+                </cauuid>
```

Comment:
Maybe a `ModelRelationField`

Reply:
The field "refid" is necessary only for migration. Then it is not being used.